src/python/library/tritonclient/grpc/aio/__init__.py

```diff
@@ -708,21 +714,35 @@ async def _request_iterator(inputs_iterator):
                     parameters=inputs["parameters"],
                 )
 
-        try:
-            response_iterator = self._client_stub.ModelStreamInfer(
-                _request_iterator(inputs_iterator),
-                metadata=metadata,
-                timeout=stream_timeout,
-                compression=_grpc_compression_type(compression_algorithm),
-            )
-            async for response in response_iterator:
+        class _ResponseIterator:
+            def __init__(self, grpc_call, verbose):
+                self._grpc_call = grpc_call
+                self._verbose = verbose
+
+            def __aiter__(self):
+                return self
+
+            async def __anext__(self):
+                response = await self._grpc_call.__aiter__().__anext__()
```

Comment:
Ahh... I was missing the `.__aiter__()` in my changes. 
Thanks for adding this!